### PR TITLE
Use in-toto types from the in-toto repo

### PR DESCRIPTION
**Pull request by rbehjati**
Ref #51

* Uses in-toto types from the in-toto repo
* Updates developer guidelines

### CODE REVIEW

**Comment by rbehjati** [comment]
Note: This does not yet remove the `Provenance` struct in our `slsa` package. To keep the PR small, I'll fix that in a follow-up PR.

**Review by aferr — APPROVED** [review]

File: experimental/auth-logic/wrappers/wrappers.go [hunk 1]
```diff
@@ -28,10 +28,10 @@ import (
 // sequence of statements) without a principal. These statements might
 // include rules or predicates. For example, the following is an unattributed
 // statement:
-// ```
+// “`
```
[the remark this answers is not in the document]
Comment (rbehjati) [reply experimental/auth-logic/wrappers/wrappers.go:31]: I am not sure. I ran `./scripts/formatting`. I was surprised too. Do you think it is wrong?

[the remark this answers is not in the document]
Comment (rbehjati) [reply experimental/auth-logic/wrappers/wrappers.go:31]: Oh! It is markdown. This change does not make sense.

[the remark this answers is not in the document]
Comment (rbehjati) [reply experimental/auth-logic/wrappers/wrappers.go:31]: Apparently you cannot have backticks on Golang docs. For code blocks it is enough to indent the code: https://go.dev/blog/godoc.

[the remark this answers is not in the document]
Comment (rbehjati) [reply experimental/auth-logic/wrappers/wrappers.go]: Removed the backticks and indented the code, similar to [doc.go](https://go.dev/src/encoding/gob/doc.go). Does it look good to you?

[the remark this answers is not in the document]
Comment (aferr) [reply experimental/auth-logic/wrappers/wrappers.go]: OK makes now. The way you have it now looks good to me.

**Comment by aferr** [comment]
> Note: This does not yet remove the `Provenance` struct in our `slsa` package. To keep the PR small, I'll fix that in a follow-up PR.

If you plan to remove the old provenance struct, should I fix the wrapper that reads from it, or is that not worthwhile since you are re-evaluating auth logic?

**Comment by rbehjati** [comment]
> If you plan to remove the old provenance struct, should I fix the wrapper that reads from it, or is that not worthwhile since you are re-evaluating auth logic?

I'll have to fix the wrappers as part of my PR, otherwise the CI checks fail. I am working on it now :)

**Pull request merged**